test/wpool_pool_SUITE.erl

```diff
@@ -324,9 +324,11 @@ random_worker(_Config) ->
         [receive
              true ->
                  true
+         after 5000 ->
+             ct:fail("Didn't receive 'true' in time")
          end
          || _ <- lists:seq(1, 20 * ?WORKERS)],
-    true = lists:all(fun(Value) -> true =:= Value end, Results),
+    true = lists:all(fun(Value) -> Value end, Results),
```

Comment:
I believe an identity function was discussed at one point in time, but alas it didn't move forward...